src/app/CMakeLists.txt

```diff
@@ -7,6 +7,17 @@ set_source_files_properties(${QBT_TS_FILES} PROPERTIES OUTPUT_LOCATION "${qBitto
 qt_add_translation(QBT_QM_FILES ${QBT_TS_FILES} OPTIONS -silent)
 configure_file("${qBittorrent_SOURCE_DIR}/src/lang/lang.qrc" "${qBittorrent_BINARY_DIR}/src/lang/lang.qrc" COPYONLY)
 
+# Custom target to update .ts files and include new strings from source files
```

Comment:
maybe move the new code above the file globing at line 5? currently it cuts in between gui and webui parts.

Reply:
Done.